Example/TestCountryPickerFramework/ViewController.swift

```diff
@@ -40,11 +40,14 @@ class ViewController: UIViewController {
     @IBAction func countryCodeButtonClicked(_ sender: UIButton) {
         switch showWithSectionsSwitch.isOn {
         case true:
-            let countryController = CountryPickerWithSectionViewController.presentController(on: self) { (country: Country) in
+            let countryController = CountryPickerWithSectionViewController.presentController(on: self) { [weak self] (country: Country) in
+                
+                guard let `self` = self else { return }
+                
```

Comment:
@Sharkesm can we change `guard let `self` = self else { return }` to `guard let self = self else { return }`  because in swift 4.x we can omit `` in self.

Reply:
Same in documentation.